cli/src/main.rs

```diff
@@ -28,52 +31,87 @@ struct Args {
     // Config file
     #[arg(short, long)]
     config_file: Option<PathBuf>,
+
+    #[arg(short, long, default_missing_value = "false")]
```

Comment:
I am very interested in this functionality as well, so just a drop by comment

Should these just be of type `bool` and remove the `default_missing_value`? Right now it panics if stdin and quiet is not provided as flags. It would also allow to type `--stdin` instead of `--stdin=true` which just is redundant IMO

I also feel like `quiet` should be default to true if `stdin` is true

Reply:
Yep. I agree. I will definitely change it to proper flags when I get around to fixing it xD